syft/frameworks/torch/tensors/interpreters/native.py

```diff
@@ -812,12 +812,14 @@ def fix_prec(self, *args, storage="auto", field_type="int100", no_wrap: bool = F
         if not kwargs.get("owner"):
             kwargs["owner"] = self.owner
 
+        # we create a copy of pointer which prevents inplace fix precision
         if self.is_wrapper:
-            self.child = self.child.fix_prec(*args, **kwargs)
+            self_copy = self.copy()
```

Comment:
Q: Is there a scenario where the ```PointerTensor``` is not a wrapper?
```
import torch
import syft

hook = syft.TorchHook(torch)
alice = syft.VirtualWorker(id="alice", hook=hook)

x = torch.Tensor([1,2,3])
ptr = x.send(alice)
ptr.child.fix_precision()
```

Reply:
Code outside this if block takes care of it (which is not modified).
`(Wrapper)>FixedPrecisionTensor>tensor([1000, 2000, 3000])` is the output, which is correct!